kano/webapp.py

```diff
@@ -94,6 +94,9 @@ def run(self):
         view.connect('close-web-view', self._close)
         view.connect('onload-event', self._onload)
 
+        language = self._get_browser_lang()
+        view.execute_script("window.navigator.userLanguage = '%s'" % language)
```

Comment:
The issue was that the language was not set correctly.
It was set to English, although I had configured my Kano with ja_JP.UTF-8.
According to the doc, the property window.navigator.language is indeed read-only.
This is why I am using window.navigator.userLanguage instead, which I was able to successfully initialise.

I was not able to spot why it was not initialised properly.
But I agree if you find a way to ensure navigator.language is properly initialised, it would be easier as we would not need to do this.